### PR TITLE
fix(entities-plugins): plugin list display incorrect scopes [KM-590]

### PR DESCRIPTION
**Pull request by 2eha0**
# Summary
Compatible with different data structures in the List and Filter APIs
[KM-590](https://konghq.atlassian.net/browse/KM-590)

[KM-590]: https://konghq.atlassian.net/browse/KM-590?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/entities/entities-plugins/src/components/PluginList.vue [hunk 1]
```diff
@@ -457,16 +457,16 @@ const errorMessage = ref<TableErrorMessage>(null)
 // transform the entity row to "Applied To" badges
 const aggregateAppliedTo = (row: EntityRow): ({ type: ViewRouteType | null, badgeText: string })[] => {
   const badges = [] as ({ type: ViewRouteType | null, badgeText: string })[]
-  if (row.route?.id) {
+  if (row.route?.id || row.route_id) {
```
Comment (Leopoldthecoder) [inline packages/entities/entities-plugins/src/components/PluginList.vue:460]: nit: add some comments as to why this change is needed

Comment (2eha0) [reply packages/entities/entities-plugins/src/components/PluginList.vue:461]: done

**Review by Leopoldthecoder — APPROVED** [review]
LGTM
